src/style.css

```diff
@@ -1,3 +1,99 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+}
+
+.roboto-medium {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  background-color: rgba(255, 255, 255, 1);
+  align-items: center;
+
+  /* padding: 0 50px; */
```

Comment:
Leaving commented-out code like this is also a bad practice
Can you please remove it here and below